packages/react-scripts/layout/views/layout.ejs

```diff
@@ -15,7 +15,10 @@
     <meta name="google-site-verification" content="VJlFQrAXTCBy0PqVi23EsGD-mpA5KaBhIt8LkrcRbP0" />
     <link href="https://edge.fscdn.org/assets/docs/fs_logo_favicon_sq.png" rel="icon" type="image/x-icon" />
     <link rel="apple-touch-icon" href="https://edge.fscdn.org/assets/components/hf/assets/img/tree-touch-icon-11024a277f1fda5735de5a9f0f4f75ca.png" />
-    <link rel="manifest" href="<%= typeof appPathOverride !== 'undefined' ? appPathOverride : appPath('') %>/manifest.json" type="application/manifest+json" crossorigin="use-credentials" />
+
+    <% if (manifestJsonExists) { %>
```

Comment:
This looks good. Your pull request description though mentioned `typeof manifestJsonExists === 'undefined'`. Is that something you are going to change this to or did you push your latest changes?